client/Angular/controllers/mainController.js

```diff
@@ -70,6 +72,20 @@ ClonageApp.controller("MainController", function($scope, $interval, userService,
                     }
                 }
             });
+        } else if (data.action === "join room observer") {
+            //make toast that lets the user join a game as an observer
+            toastr.warning('<div id="toaster">Game already in progress<br> Click here to watch as an Observer</div>', {
+                allowHtml: true,
+                showCloseButton: true,
+                timeOut: 10000,
```

Comment:
maybe set timeout to null so it doesn't dissapear ? if players aren't payig attention might loose the notification?